Source/Core/DolphinQt2/TAS/IRWidget.h

```diff
@@ -0,0 +1,38 @@
+// Copyright 2018 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QDialog>
+
+#include "Common/CommonTypes.h"
+
+class IRWidget : public QWidget
+{
+  Q_OBJECT
+public:
+  explicit IRWidget(QWidget* parent);
+
+signals:
+  void ChangedX(u16 x);
+  void ChangedY(u16 y);
+
+public slots:
+  void SetX(u16 x);
+  void SetY(u16 y);
+
+protected:
+  void paintEvent(QPaintEvent* event) override;
+  void mousePressEvent(QMouseEvent* event) override;
+  void mouseMoveEvent(QMouseEvent* event) override;
+  void handleMouseEvent(QMouseEvent* event);
+
+public:
+  static const u16 m_max_x = 1023;
```

Comment:
This shouldn't have a ``m_`` prefix as it isn't a member of the class.
Also try using ``constexpr``, that might fix the issue with the OSX buildbot.